packages/core/src/database/index.ts

```diff
@@ -713,11 +716,30 @@ EXECUTE PROCEDURE "${namespace.schema}".${notification};`,
           }
 
           // Note: it is an invariant that checkpoints.length > 0;
-          const revertCheckpoint = min(
-            ...checkpoints.map((c) => c.safeCheckpoint),
-          );
+          switch (ordering) {
+            case "omnichain": {
+              const revertCheckpoint = min(
+                ...checkpoints.map((c) => c.safeCheckpoint),
+              );
 
-          await revert(tx, { checkpoint: revertCheckpoint, tables, ordering });
+              await revert(tx, {
```

Comment:
Would rather have a new `crashRecovery` action that just reverts every row from the reorg table. I could see the multiple reverts in multichain ordering causing us trouble.